src/style.css

```diff
@@ -1 +1,56 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf), url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 300;
+}
+
+html {
+  margin: 0;
+}
+
+.search-form {
+  margin-top: 20px;
+}
+
+.search-input {
+  box-sizing: border-box;
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  font-family: Avenir, sans-serif;
```

Comment:
You should include fallback fonts in the `font-family` property to ensure text is displayed with a default font in case the custom font fails to load. For example: `font-family: Avenir, Arial, sans-serif;`.